Dockerfile.riot

```diff
@@ -1,4 +1,10 @@
-ARG riot_version=latest
+ARG riot_version=v1.5.15
```

Comment:
usually we leave this as latest since it's overwritten when we use make, to avoid duplicating the version number in two places